packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.tsx

```diff
@@ -161,7 +161,7 @@ function SpanBar(props: TInnerProps) {
     <div
       className={cx(styles.wrapper, className)}
       onClick={onClick}
-      onMouseOut={setShortLabel}
+      onMouseLeave={setShortLabel}
```

Comment:
Out of curiosity why this had to be changed?

Reply:
I had a problem with the tooltip not showing. And after changing that to onMouseLeave it showed up.

From [w3schools](https://www.w3schools.com/jsref/event_onmouseleave.asp)
> Tip: The onmouseleave event is similar to the onmouseout event. The only difference is that the onmouseleave event does not bubble (does not propagate up the document hierarchy).